runtime/runtime/src/verifier.rs

```diff
@@ -124,32 +124,43 @@ pub fn verify_and_charge_tx_ephemeral(
     transaction_cost: &TransactionCost,
     block_height: Option<BlockHeight>,
     current_protocol_version: ProtocolVersion,
+    ephemeral_state: Option<(Account, AccessKey)>,
```

Comment:
I think you could take a `&mut` for this here.  And then you do not have to keep cloning it each time you call this function.